examples/react-instantsearch-hooks/src/components/Autocomplete.tsx

```diff
@@ -228,13 +227,7 @@ export function Autocomplete({
           });
         }
       },
-      renderer: {
-        createElement,
-        Fragment,
-      },
-      render({ children }, root) {
-        render(children as ReactElement, root);
-      },
+      renderer: { createElement, Fragment, render: render as Render },
```

Comment:
why is the cast needed? is the type too narrow to accept react's render?

Reply:
Yes, but it's assignable to `Render` so the cast is valid.